### PR TITLE
Added package installation instructions

### PR DESCRIPTION
**Pull request by soramikan**
Thanks for making a great plugin!
However, because the README did not describe how to install the package, we had to find it through PyPI.
Let's add the package installation instructions to the README for people like me later on!
Thanks.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,20 +16,25 @@ hands off the serving of the file to a web server, like Nginx.
...
+2. Add "protected_media" to your INSTALLED_APPS setting like this:
 ```python
 INSTALLED_APPS = [
     ...
     'protected_media.apps.ProtectedMediaConfig',
 ]
 ```
 
-2. Include the URLconf in your project urls.py like this::
+1. Include the URLconf in your project urls.py like this::
```
Comment (cobusc) [inline README.md]: Please change to "3" and remove duplicated colon at the end of the line

File: README.md [hunk 2]
```diff
@@ -16,20 +16,25 @@ hands off the serving of the file to a web server, like Nginx.
...
 ```
 path('protected/', include('protected_media.urls')),
 ```
 
-3. Add the following settings to `settings.py` if the defaults need to be tweaked:
+1. Add the following settings to `settings.py` if the defaults need to be tweaked:
```
Comment (cobusc) [inline README.md]: Please change to 4.

File: README.md [hunk 3]
```diff
@@ -38,7 +43,7 @@ PROTECTED_MEDIA_LOCATION_PREFIX = "/internal"  # Prefix used in nginx config
 PROTECTED_MEDIA_AS_DOWNLOADS = False  # Controls inclusion of a Content-Disposition header
 ```
 
-4. Use the new field classes in your models:
+1. Use the new field classes in your models:
```
Comment (cobusc) [inline README.md]: Please change to 5

**Review by cobusc — CHANGES_REQUESTED** [review]
Hi. Thanks for the PR!

Can you please make the suggested changes, then I'll merge.

**Comment by soramikan** [comment]
Thanks for the quick review!
Sorry, it seems my editor's completion feature worked incorrectly.
Fixed it.

**Review by cobusc — APPROVED** [review]

**Pull request merged**